.taskcluster.yml

```diff
@@ -0,0 +1,315 @@
+                                              - index.project.mobile.${project}.v2.branch.${head_branch}.revision.${head_sha}.taskgraph.cron.${ownTaskId}
+                          scopes:
+                              $if: 'tasks_for == "github-push"'
+                              then:
+                                  # `https://` is 8 characters so, ${repoUrl[8:]} is the repository without the protocol.
+                                  - 'assume:repo:${repoUrl[8:]}:branch:${short_head_branch}'
+                              else:
+                                  $if: 'tasks_for == "github-pull-request"'
+                                  then:
+                                      - 'assume:repo:github.com/${event.pull_request.base.repo.full_name}:pull-request'
+                                  else:
+                                      $if: 'tasks_for == "github-release"'
+                                      then:
+                                          - 'assume:repo:${repoUrl[8:]}:release'
+                                      else:
+                                          $if: 'tasks_for == "action"'
+                                          then:
+                                              # when all actions are hooks, we can calculate this directly rather than using a variable
+                                              - '${action.repo_scope}'
+                                          else:
+                                              - 'assume:repo:${repoUrl[8:]}:cron:${cron.job_name}'
+
+                          requires: all-completed
+                          priority: lowest
+                          retries: 5
+
+                          payload:
+                              env:
+                                  # run-task uses these to check out the source; the inputs
+                                  # to `mach taskgraph decision` are all on the command line.
+                                  $merge:
+                                      - MOBILE_BASE_REPOSITORY: '${baseRepoUrl}'
+                                        MOBILE_HEAD_REPOSITORY: '${repoUrl}'
+                                        MOBILE_HEAD_REF: '${head_branch}'
+                                        MOBILE_HEAD_REV: '${head_sha}'
+                                        MOBILE_HEAD_TAG: '${head_tag}'
+                                        MOBILE_REPOSITORY_TYPE: git
+                                        TASKGRAPH_BASE_REPOSITORY: https://hg.mozilla.org/ci/taskgraph
+                                        TASKGRAPH_HEAD_REPOSITORY: https://hg.mozilla.org/ci/${taskgraph.branch}
+                                        TASKGRAPH_HEAD_REV: ${taskgraph.revision}
+                                        TASKGRAPH_REPOSITORY_TYPE: hg
+                                        REPOSITORIES: {$json: {mobile: "Firefox for IOS", taskgraph: "Taskgraph"}}
```

Comment:
Oh, great catch! My tests didn't catch this issue because taskgraph doesn't schedule any tasks yet. It would have become a problem later. 

Fixed. Thank you 🙂